src/cljs/rems/application.cljs

```diff
@@ -30,13 +32,13 @@
             [rems.actions.vote :refer [vote-action-button vote-form votes-summary]]
             [rems.application-list :as application-list]
             [rems.administration.duo :refer [duo-field duo-info-field]]
-            [rems.common.application-util :refer [accepted-licenses? can-see-everything? form-fields-editable? get-member-name is-handler? is-handling-user?]]
+            [rems.common.application-util :as application-util :refer [accepted-licenses? can-see-everything? form-fields-editable? get-member-name is-handler? is-handling-user?]]
```

Comment:
This change is perhaps not needed?

Maybe it would make sense overall to use `application-util` stuff with the ns in the future, and not refer. But using both at the same time perhaps doesn't make sense.